.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: python
+
+python:
+   
+  - "3.7"
+
+services:
+  - postgresql
+
+env:
+  - DJANGO_VERSION=2.0.7
+
+before_script:
+  - psql -c 'create database test_db;' -U postgres
+ 
+install:
+  - pip install -r requirements.txt
+
+script:
+  - coverage run --source='.' manage.py test 
```

Comment:
Do we have coverage installed?

Reply:
Yes, it's installed.

Reply:
@AGMETEOR I don't see it in requirements.txt 🤔